cookbooks/aws-parallelcluster-slurm/files/default/head_node_slurm/slurm/templates/slurm_parallelcluster_cgroup.conf

```diff
@@ -0,0 +1,7 @@
+# slurm_parallelcluster.conf is managed by the pcluster processes.
+# Do not modify.
+# Please add user-specific slurm configuration options in cgroup.conf
+
+{% if scaling_config.EnableMemoryBasedScheduling %}
+ConstrainRAMSpace=yes
```

Comment:
Because `no` is already the default value for that option. But at least setting it explicitly would protect us from a (quite unlikely) change in default value coming from SchedMD. Addressed.